include/stdshims.h

```diff
@@ -0,0 +1,19 @@
+//! Shims for std:: functions that aren't available in the current C++ versions
+//! we target.
+
+#pragma once
```

Comment:
Not critical, but some compilers may not understand this directive. How about using macros for this purpose(you may use macros and pragma directives simultaneously)?

Reply:
Never witnessed it in person, but okay - #ifdef-style include guards seem to be the norm for LLMS so I've updated to use those.